doc/ipython-notebooks/classification/MKL.ipynb

```diff
@@ -694,30 +685,30 @@
     "feats_rem = sg.features(Xrem)             \n",
     "labels_rem = sg.MulticlassLabels(Yrem)\n",
     "\n",
-    "kernel = sg.CombinedKernel()\n",
+    "kernel = sg.kernel(\"CombinedKernel\")\n",
     "feats_train = sg.CombinedFeatures()\n",
     "feats_test = sg.CombinedFeatures()\n",
     "\n",
     "#append gaussian kernel\n",
     "subkernel = sg.kernel(\"GaussianKernel\", log_width=np.log(15))\n",
     "feats_train.append_feature_obj(feats)\n",
     "feats_test.append_feature_obj(feats_rem)\n",
-    "kernel.append_kernel(subkernel)\n",
+    "kernel.add(\"kernel_array\", subkernel)\n",
     "\n",
     "#append PolyKernel\n",
     "feats  = sg.features(Xtrain)\n",
     "subkernel = sg.kernel('PolyKernel', degree=10, c=2)\n",
     "feats_train.append_feature_obj(feats)\n",
     "feats_test.append_feature_obj(feats_rem)\n",
-    "kernel.append_kernel(subkernel)\n",
+    "kernel.add(\"kernel_array\", subkernel)\n",
     "\n",
     "kernel.init(feats_train, feats_train)\n",
     "\n",
-    "mkl = sg.MKLMulticlass(1.2, kernel, labels)\n",
+    "mkl = sg.machine(\"MKLMulticlass\", C=1.2, kernel=kernel, \n",
+    "                 labels=labels, mkl_eps=0.001, mkl_norm=1)\n",
     "\n",
-    "mkl.set_epsilon(1e-2)\n",
-    "mkl.set_mkl_epsilon(0.001)\n",
-    "mkl.set_mkl_norm(1)\n",
+    "# FIX ME: need a way to interact with SVM\n",
+    "# mkl.set_epsilon(1e-2)\n",
```

Comment:
@karlnapf @vigsterkr need to come up with a way to handle these situations. Would we .get() the SVM and manipulate it?

Reply:
it is commented out here, so it didnt work before I guess? :)
Is this something that would need to be applied recursively? I.e. all svms need to have set the epsilon. Or is it something of the MKL class?

Reply:
isn't there only a single SVM? In any case, what happens is that `MulticlassSVM::set_epsilon` is called, which casts the underlying `m_machine` to SVM and calls `set_epsilon` on that

Reply:
I see. Maybe
`mkl.get('svm').put('epsilon')`

Since we are using oop design, and the user provides an object, why not do that?

Or we somehow take over the SVM parameters with the MKL class so that we can use "put"?

Reply:
Yup, that's what I did, just get.("machine").put(), it keeps the interface cleaner imo